Beagbarreto/react-certification-2020
# Challenge five

## WHAT THIS PR DOES
Mini Challenge 5, implemented routes, created side menu and login and logout button.
Added feature to add and remove favorite videos and implemented mock authentication.
Includes some testing.
## HOW TO TEST THIS PR
Open the menu from the burger icon and click on the different options, should show different screens.
Login and out to see the different features available for authenticated users.
On video card, click on the star to fav a video and with the side menu go to favorite pages to view the list of videos.
## ADDITIONAL INFORMATION
Config file might have to be added externally since it doesn't upload here, in order to protect api key.
## WHAT DOES "DONE" MEAN?
This PR has been:
- reviewed again by someone else (could be current review)
## COMMIT HISTORY
